pyabc/storage/history.py

```diff
@@ -209,9 +209,12 @@ def _find_latest_id(self):
         """
         abcs = self._session.query(ABCSMC).all()
         if len(abcs) > 0:
-            return abcs[-1].id
+            for abc in reversed(abcs):
+                if abc.end_time is not None:
```

Comment:
hm, the end_time is not filled if an abc run is cancelled manually. maybe:

```python
if len(abc.populations) > 0:
    return abc.id
```
but not sure if that is valid code, so check first

Reply:
please also update the docs in line 205ff

Reply:
if `abc.populations` contain> 0 but still, the run didn't finish correctly. Then we still can't extract data from the database. 
`df, _ = history.get_distribution(m=m)` will be empty.

Reply:
I think as soon as a population has been written to database, we have to consider it a successful iteration, so len(abc.populations) > 0 would actually be the correct criterion. Usually, users will have only one or two abc analyses in the database, and use different databases for different runs, so usually this should not lead to problems. If one uses one database for multiple runs, it makes sense to note down the id used for a specific run, so that one can then explicitly call ``pyabc.History(..., _id=id)``.